File: C/Iniciante/1534/1534.c
#include <stdio.h>
 
int main(){
 
    int N, matriz[70][70], i, j;
 
    while(scanf("%d", &N) != EOF){

        for(i = 0; i < N; i++){
            for(j = 0; j < N; j++) {
                if((i + j) == (N - 1)) {
                    matriz[i][j] = 2;
                } 
                else if (i == j){
                    matriz[i][j] = 1;
                }
                else {
                    matriz[i][j] = 3;
                }
            }
        }
 
        for(i = 0; i < N; i++){
            for(j = 0; j < N; j++) {
                printf("%d", matriz[i][j]);
            }
            printf("\n");
        }
    };

    return 0;
}

File: Python/1534.py
# Problema 1534 - URI - Iniciante - Nível 1

# Loop responsável pelos casos de testes
while True:
  try:
    # Orde da matrizes
    ordem = int(input())
    # Criação da matriz e leitura dos números
    matriz = []
    for i in range(ordem):
      # Criando as demais dimensões
      auxiliar = []
      for j in range(ordem):
        if(i + j == ordem - 1):
          auxiliar.append(2)
        elif(i == j):
          auxiliar.append(1)
        else:
          auxiliar.append(3)
      matriz.append(auxiliar)
    # Imprimindo a matriz
    for i in range(ordem): 
      for j in range(ordem):
        print(matriz[i][j], end = '')
      print() 
  # Caso de exceção chegar ao EOF
  except:
    break